### ISSUE TITLE
Flask server working fine but getting response "null"

### ISSUE THREAD

**Issue by SatyaDewangan05** [Priyanshu-hawk/ChatGPT-unofficial-api-selenium#3]
Flask server is working fine and chatgpt is getting prompt and generating result but i'm getting response "null"

```
{
    "response": null
}
```

**Comment by SatyaDewangan05** [comment]
I just reviewed the code and found that it's only working in light theme not in dark theme. This is happening due to the classname difference caused by the using different theme. I just simply need to replace this on line no 97 on api_backend.py

```
response_xpath = "//*[@class='markdown prose w-full break-words dark:prose-invert light']"
```

```
response_xpath = "//*[@class='markdown prose w-full break-words dark:prose-invert dark']"
```

**Comment by Priyanshu-hawk** [comment]
Thankyou @SatyaDewangan05 for raising this issue, really appreciate.

I've just added a fix for this @ commit 44c23298ff9b8b0039684173cbbf9f2002d906e8.
you can pull the latest code and switch to `devAPI` branch and pls check if it is working for you or not.

Regards.
